### PR TITLE
IA update - creates Asset Hub page for Protocol section

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR does the following:
- creates Asset Hub page for `polkadot-protocol/architecture/system-chains/asset-hub.md`
- uses information from: #21 and #59 plus online resources

### CODE REVIEW

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 1]
```diff
@@ -1,4 +1,204 @@
...
----
\ No newline at end of file
+description: Learn about Asset Hub in Polkadot, managing on-chain assets, foreign asset integration, and using XCM for cross-chain asset transfers.
+---
+
+# Asset Hub
+
+## Introduction
+
+The Asset Hub is a critical component in the Polkadot ecosystem, enabling the management of fungible and non-fungible assets across the network. Since the relay chain focuses on maintaining security and consensus without direct asset management, Asset Hub provides a streamlined platform for creating, managing, and using on-chain assets in a fee-efficient manner. This guide outlines the core features of Asset Hub, including how it handles asset operations, cross-chain transfers, and asset integration using XCM, as well as essential tools like API Sidecar and TxWrapper for developers working with on-chain assets.
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:10]: Perhaps we should link these tools appropriately?

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 2]
```diff
@@ -1,4 +1,204 @@
...
+- The total supply
+- The number of accounts holding the asset
+- Sufficiency for account existence - whether the asset alone is enough to maintain an account without a native token balance
+- The metadata of the asset, including its name, symbol, and the number of decimals for representation
+
+Some assets can be regarded as sufficient to maintain an account's existence, meaning that users can create accounts on the network without needing a native token balance (i.e., no existential deposit required). Developers can also set minimum balances for their assets. If an account's balance drops below the minimum, the balance is considered dust and may be cleared.
+
+## Assets Pallet
+
+The Polkadot SDK's Assets pallet is a powerful tool for creating and managing fungible asset classes with a fixed supply. It offers a secure and flexible way to issue, transfer, freeze, and destroy assets. The pallet supports various operations and includes permissioned and non-permissioned functions to cater to simple and advanced use cases.
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:37]:
```suggestion
The Polkadot SDK's Assets pallet is a powerful module designiated for creating and managing fungible asset classes with a fixed supply. It offers a secure and flexible way to issue, transfer, freeze, and destroy assets. The pallet supports various operations and includes permissioned and non-permissioned functions to cater to simple and advanced use cases.
```

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 3]
```diff
@@ -1,4 +1,204 @@
...
+
+For full details, see the [assets pallet repository](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/assets){target=\_blank}, which contains reference documentation and examples of how to interact with the pallet.
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:39]: I would recommend this as the link instead, reads a bit better: https://paritytech.github.io/polkadot-sdk/master/pallet_assets/index.html

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 4]
```diff
@@ -1,4 +1,204 @@
...
+The Assets pallet provides a broad interface for managing fungible assets. Some of the main dispatchable functions include:
+
+- **`create()`** - create a new asset class by placing a deposit, applicable when asset creation is permissionless
+- **`issue()`** - mint a fixed supply of a new asset and assign it to the creator's account
+- **`transfer()`** - transfer a specified amount of an asset between two accounts
+- **`approve_transfer()`** - approve a non-custodial transfer, allowing a third party to move assets between accounts
+- **`destroy()`** - destroy an entire asset class, removing it permanently from the chain
+- **`freeze()`** and **`thaw()`** - administrators or privileged users can lock or unlock assets from being transferred
+
+For a full list of dispatchable and privileged functions, see the [Call Enum Rust docs](https://docs.rs/pallet-assets/latest/pallet_assets/pallet/enum.Call.html){target=\_blank}.
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:62]:
```suggestion
For a full list of dispatchable and privileged functions, see the [dispatchables located in the Rust Docs](https://paritytech.github.io/polkadot-sdk/master/pallet_assets/pallet/dispatchables/index.html){target=\_blank}.
```

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 5]
```diff
@@ -1,4 +1,204 @@
...
+
+### Public Functions
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:64]: I'm not sure if we should use the term "public functions" - maybe "queryable", or something along those lines? They're functions that query chain state / storage mappings

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 6]
```diff
@@ -1,4 +1,204 @@
...
+
+The API supports various asset operations, such as paying transaction fees with non-native tokens and managing asset liquidity.
+
+### Parachain Node
+
+To fully leverage the Asset Hub's functionality, developers will need to run a system parachain node. Setting up an Asset Hub node allows users to interact with the parachain in real time, syncing data and participating in the broader Polkadot ecosystem. Guidelines for setting up an [Asset Hub node](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus#asset-hub-){target=\_blank} are available in the Parity documentation.
+
+Using these integration tools, developers can manage assets seamlessly and integrate Asset Hub functionality into their applications, leveraging Polkadot's powerful infrastructure.
+
+## XCM Transfer Monitoring
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:168]: This feels like it should be in another page, but in the future maybe we can consider moving out as it's out of scope for this PR.

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 7]
```diff
@@ -1,4 +1,204 @@
...
+
+- **Transfers** - once registered in the Asset Hub, foreign assets can be transferred between accounts, just like native assets. Users can also send these assets back to their originating blockchain if supported by the relevant cross-chain messaging mechanisms
+
+## Integration
+
+Asset Hub supports a variety of integration tools that make it easy for developers to manage assets and interact with the blockchain in their applications. The tools and libraries provided by Parity Technologies enable streamlined operations, such as querying asset information, building transactions, and monitoring cross-chain asset transfers.
+
+Developers can integrate Asset Hub into their projects using these core tools:
+
+### API Sidecar
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:128]: For these tools - is it a good idea to just mention them here, when beyond AssetHub, you could use them for any parachain?

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 8]
```diff
@@ -1,4 +1,204 @@
...
+- **Asset balances** - view the balance of a particular asset on Asset Hub
+
+Public instances of API Sidecar connected to Asset Hub are available, such as:
+
+- [Polkadot Asset Hub Sidecar](https://polkadot-asset-hub-public-sidecar.parity-chains.parity.io/){target=\_blank}
+- [Kusama Asset Hub Sidecar](https://kusama-asset-hub-public-sidecar.parity-chains.parity.io/){target=\_blank}
+
+These public instances are primarily for ad-hoc testing and quick checks.
+
+### TxWrapper
```
Comment (CrackTheCode016) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:142]: Same for TxWrapper

**Review by CrackTheCode016 — COMMENTED** [review]
Besides some minor comments on the content and arrangment, LGTM!

File: polkadot-protocol/architecture/system-chains/asset-hub.md (hunk 1 above)
Comment (dawnkelly09) [reply polkadot-protocol/architecture/system-chains/asset-hub.md:10]: I will link here to their sections further down in the document. The individual sections have links to the source code, etc. Cleaner this way, I think.

File: polkadot-protocol/architecture/system-chains/asset-hub.md (hunk 7 above)
Comment (dawnkelly09) [reply polkadot-protocol/architecture/system-chains/asset-hub.md:128]: As someone new to the eco, this reads to me like API Sidecar is a tool I can use with any Kusama or Polkadot node, so it makes sense to me. I feel the same about the TxWrapper part. I will defer to you and @nhussein11, though. Just let me know if you want me to remove these or do something else.

File: polkadot-protocol/architecture/system-chains/asset-hub.md (hunk 8 above)
Comment (dawnkelly09) [reply polkadot-protocol/architecture/system-chains/asset-hub.md:142]: See above re: API sidecar

**Comment by dawnkelly09** [comment]
@CrackTheCode016 see what you think of the small tweaks, then it should be ready for Erin. Thanks!

**Review by CrackTheCode016 — APPROVED** [review]
LGTM, nice work, reads really well!

File: polkadot-protocol/architecture/system-chains/asset-hub.md [hunk 9]
```diff
@@ -1,4 +1,203 @@
...
+
+### Key Features
+
+Key features of the Assets pallet include:
+
+- **Asset issuance** - allows the creation of a new asset, where the total supply is assigned to the creator's account
+- **Asset transfer** - enables transferring assets between accounts while maintaining a balance in both accounts
+- **Asset freezing** - prevents transfers of a specific asset from one account, locking it from further transactions
+- **Asset destruction** - allows accounts to burn or destroy their holdings, removing those assets from circulation
+- **Approval-based transfers** - a non-custodial mechanism to enable one account to approve a transfer of assets on behalf of another
```
Comment (themacexpert) [inline polkadot-protocol/architecture/system-chains/asset-hub.md:49]: Do you want to give this the same name you give it down below?
**Non-Custodial Transfers (Approval API)**

**Review by themacexpert — APPROVED** [review]
lgtm

**Review by eshaben — APPROVED** [review]
LGTM!

**Review by eshaben — APPROVED** [review]

**Pull request merged**